deployment/environment/memory/environment.go

```diff
@@ -101,6 +101,9 @@ func generateMemoryChain(t *testing.T, inputs map[uint64]EVMChain) map[uint64]de
 
 func NewNodes(t *testing.T, logLevel zapcore.Level, chains map[uint64]deployment.Chain, numNodes, numBootstraps int, registryConfig deployment.CapabilityRegistryConfig) map[string]Node {
 	nodesByPeerID := make(map[string]Node)
+	if numNodes+numBootstraps == 0 {
```

Comment:
This lets you just omit the nodes and you'll get a chain only env which is pretty useful